### PR TITLE
Fix a typos

### PR DESCRIPTION
**Pull request by yanghaichao12**
What type of PR is this?
/kind cleanup

What this PR does / why we need it:
Fix a typos

Which issue(s) this PR fixes (optional, in fixes #(, fixes #<issue_number>, ...) format, will close the issue(s) when PR gets merged):
Fixes #

Special notes for your reviewer:

Does this PR introduce a user-facing change?:

NONE

### CODE REVIEW

**Comment by yanghaichao12** [comment]
@jiayingz could you help to merge this pr?thx a lot!

**Comment by nikhita** [comment]
@yanghaichao12 While this PR is definitely correct, if it's possible can you club similar smaller typos into this PR? Would be good to club various typos in a single PR instead of individual smaller ones. :)

**Comment by yanghaichao12** [comment]
@nikhita hi,thank for your suggestion!i would do that later.

**Comment by yanghaichao12** [comment]
@nikhita done :)

**Comment by yanghaichao12** [comment]
@RenaudWasTaken hi，i'v find other typos and add to this pr. would you do me a favor to check it again?
thanks a lot!

File: pkg/kubelet/cm/helpers.go [hunk 1]
```diff
@@ -21,7 +21,7 @@ import (
 	evictionapi "k8s.io/kubernetes/pkg/kubelet/eviction/api"
 )
 
-// hardEvictionReservation returns a resourcelist that includes reservation of resources based on hard eviction thresholds.
+// hardEvictionReservation returns a resource list that includes reservation of resources based on hard eviction thresholds.
```
Comment (nikhita) [inline pkg/kubelet/cm/helpers.go]: I think `ResourceList` would be more appropriate here.

Comment (yanghaichao12) [reply pkg/kubelet/cm/helpers.go]: Although this function returns a 'v1.ResoureList' object, this comment seems to emphasize the data structure of the list, 
So I think the lowercase 'resourcelist' is not suitable here......

**Comment by yanghaichao12** [comment]
i can't figure out ci error,so close this pr and open another one:#72850

**Pull request closed without merging**